soccer/src/soccer/radio/packet_convert.cpp

```diff
@@ -110,6 +135,36 @@ void status_to_proto(const RobotStatus& status, Packet::RadioRx* proto) {
             break;
     }
 }
+void status_to_ros(const RobotStatus& status, rj_msgs::msg::RobotStatus* msg) {
+    rj_convert::convert_to_ros(status.timestamp, &(msg->timestamp));
+    msg->robot_id = status.shell_id;
+    msg->battery_voltage = status.battery_voltage;
+    msg->motor_errors = status.motors_healthy;
+    msg->has_ball_sense = status.has_ball;
+    msg->kicker_charged = status.kicker == RobotStatus::KickerState::kCharged;
+    msg->kicker_healthy = status.kicker != RobotStatus::KickerState::kFailed;
+    msg->fpga_error = !status.fpga_healthy;
+
+    // TODO(Kyle): Handle encoders.
```

Comment:
Should the python code be able to use the encoder data from this message? It defiantly doesn't currently. What type of data would be going into the RobotStatus?